specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2018-11-20/guestconfiguration.json

```diff
@@ -861,16 +861,8 @@
           ]
         },
         "allowModuleOverwrite": {
-          "type": "string",
-          "description": "If true - new configurations downloaded from the pull service are allowed to overwrite the old ones on the target node. Otherwise, false",
-          "x-ms-enum": {
-            "name": "allowModuleOverwrite",
-            "modelAsString": true
-          },
-          "enum": [
-            "True",
-            "False"
-          ]
+          "type": "boolean",
+          "description": "If true - new configurations downloaded from the pull service are allowed to overwrite the old ones on the target node. Otherwise, false"
```

Comment:
Same question as in above api-version, because all of other properties are read-only, should this one be read-only also?

Reply:
Fixed. Removed the read-only.
This was by mistake.